node/lib/util/cherry_pick_util.js

```diff
@@ -755,21 +757,57 @@ exports.rewriteCommit = co.wrap(function *(repo, commit) {
         (0 !== Object.keys(changes.simpleChanges).length || 0 !== nChanges)) {
         result.newMetaCommit =
                             yield SubmoduleRebaseUtil.makeCommit(repo, commit);
+        }
+
+    if (result.errorMessage === null) {
+        // Run post-commit hook as regular git.
+        yield Hook.execHook(repo, "post-commit");
     }
     return result;
 });
 
+const pickRemainingCommits = co.wrap(function*(metaRepo, seq) {
+    const commits = seq.commits;
+    let result;
+    for (let i = seq.currentCommit; i < commits.length; ++i) {
+        const id = commits[i];
+        const commit = yield metaRepo.getCommit(id);
+        console.log(`Cherry-picking commit ${colors.green(id)}.`);
```

Comment:
don't you need to check if there are conflicts before applying the rewriting the commit?

Reply:
see above where it checks status.isConflicted